airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/dest_state_lifecycle_manager/DestStateLifecycleManager.java

```diff
@@ -28,6 +31,8 @@ public interface DestStateLifecycleManager {
 
   /**
    * Moves any tracked state messages that are currently pending to flushed.
+   *
+   * @Deprecated
```

Comment:
I generally like to add a message for why this method is deprecated, but totally reasonable to say that you never want to look at this file again